### ISSUE TITLE
Android App

### ISSUE THREAD

**Issue by arjung24** [YamenSharaf/real-debrid#1]
Is there any plan to make android app?

**Comment by YamenSharaf** [comment]
Hi @arjung24. Thanks for taking a look at my project.

There's already a brilliant open-source app for Android for managing your Real-Debrid account. Check out [Unchained](https://f-droid.org/en/packages/com.github.livingwithhippos.unchained/).

**Comment by arjung24** [comment]
> Hi @arjung24. Thanks for taking a look at my project.
> 
> There's already a brilliant open-source app for Android for managing your Real-Debrid account. Check out [Unchained](https://f-droid.org/en/packages/com.github.livingwithhippos.unchained/).

Yes. Right now i am using unchained. It's amazing app for realdebrid.
I just wanted to try yours as android app. Hope in future you will make it available for us. Thanks brother.
